com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -388,33 +402,32 @@ internal static void PerformDilation()
                     // Make sure unloading happens.
                     prv.PerformPendingOperations();
 
-                    Dictionary<string, bool> assetCleared = new Dictionary<string, bool>();
-                    // Put back cells
+                    // Commit cell changes
+                    HashSet<string> assetsCommitted = new HashSet<string>();
                     foreach (var cell in dilatedCells)
                     {
                         foreach (var sceneData in perSceneDataList)
                         {
                             var asset = sceneData.GetCurrentStateAsset();
 
-                            if (asset == null) continue;
+                            // How would this happen?
+                            if (asset == null)
+                                continue;
 
                             var assetPath = asset.GetSerializedFullPath();
-                            bool valueFound = false;
-                            if (!assetCleared.TryGetValue(assetPath, out valueFound))
-                            {
-                                asset.cells.Clear();
-                                assetCleared.Add(asset.GetSerializedFullPath(), true);
-                                UnityEditor.EditorUtility.SetDirty(asset);
-                            }
-
                             if (cell2Assets[cell.index].Contains(assetPath))
                             {
-                                asset.cells.Add(cell);
+                                if (!assetsCommitted.Contains(assetPath))
+                                {
+                                    WritebackModifiedCellsData(asset);
```

Comment:
Because of the way data storage is aliased, dilation now writes directly into the backing storage and we just re-commit that to disk.